fleet_operations/models/fleet_service.py

```diff
@@ -29,6 +30,8 @@ def unlink(self):
 
 
 class FleetVehicleLogServices(models.Model):
+    _inherit = 'fleet.vehicle.log.services'
```

Comment:
def copy(self, default=None):
         if not default:
             default = {}
         raise Warning(_('You can\'t duplicate record!'))
return super(ServiceCategory, self).copy(default=default)

>>> raise pa6i return?? na jaamyu